scripts/cluster_providers/kind_delete_cluster

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+EXISTING_CLUSTER=KIND_$(echo $CLUSTER_VERSION | sed -e "s/\./\_/g")
```

Comment:
@pedjak currently script does look for `KIND_$CLUSTER_VERSION` env variable and skips deletion if cluster is already exist otherwise deletes the cluster based on the cluster name. any specific reason to change the logic or am I missing something ?

Reply:
it check for the existence of the env variable, but it does not check its value. Hence, it is going to remove the cluster even if env var is set to `foo`, but we get `bar` as the script arg. We should also pass cluster version as the script arg, not via `CLUSTER_VERSION` (btw, this variable will be renamed to plural and contain potentially the list of versions)

Reply:
@pedjak could you please check the updated version of this file ? I think you would see expected changes there

Reply:
> could you please check the updated version of this file ? I think you would see expected changes there

Nope, my comments still apply.